packages/cli/commands/create/api-sample.js

```diff
@@ -37,14 +36,13 @@ module.exports = {
         return;
       }
     }
-    const downloadSpinner = ora(i18n(`${i18nKey}.loading.apiSamples`));
```

Comment:
This download happens so quickly that there's no point in us putting a loading spinner here.